### PR TITLE
New deprecated flags

### LINKED ISSUE #363: skupper-cli update: remove --version flag
remove the --version flag. We are keeping only "skupper version" command (which is already implemented)

For this, just removing the rootCmd.Version flag assignment is required.

### LINKED ISSUE #364: Deprecate "skupper list-exposed" command
For this, the replacement command must be implemented!
image

### LINKED ISSUE #366: skupper global flags updates/renames
image

image

### PR DESCRIPTION
**Pull request by nicob87**
Closes #366 
Closes #363
Closes #364

skupper-cli update: 
remove --version flag
remove --enable-proxy-controller flag
added --router-mode [interior|edge] and deprecated "--edge"

### CODE REVIEW

File: cmd/skupper/skupper.go [hunk 1]
```diff
@@ -209,6 +221,12 @@ installation that can then be connected to other skupper installations`,
 	cmd.Flags().StringVarP(&routerCreateOpts.Password, "console-password", "", "", "Skupper console user. Valid only when --console-auth=internal")
 	cmd.Flags().BoolVarP(&routerCreateOpts.ClusterLocal, "cluster-local", "", false, "Set up skupper to only accept connections from within the local cluster.")
 
+	cmd.Flags().BoolVarP(&routerCreateOpts.IsEdge, "edge", "", false, "Configure as an edge")
+	f := cmd.Flag("edge")
+	f.Deprecated = "This flag is deprecated, use --router-mode [interior|edge]"
+	f.Hidden = true
+	cmd.Flags().StringVarP(&routerMode, "router-mode", "", "edge", "Skupper router-mode")
```
Comment (ajssmith) [inline cmd/skupper/skupper.go:228]: iirc, router-mode should default to "interior" in anticipation of edge flag going away.

Suggest the router-mode arg be verified as one of "interior" or "edge" to detect bad value

Comment (nicob87) [reply cmd/skupper/skupper.go]: all true!, fixed in the last commit.

Thanks!

**Review by ajssmith — APPROVED** [review]
lgtm, thanks!

**Review by fgiorgetti — APPROVED** [review]
LGTM

**Pull request merged**